edx_django_utils/plugins/README.rst

```diff
@@ -21,6 +24,27 @@ While Django+Python already support dynamic installation of components/apps,
 they do not have out-of-the-box support for plugin apps that auto-install
 into a containing Django project.
 
+.. _decision (ADR) for moving this to edx-django-utils: https://github.com/openedx/edx-django-utils/blob/master/docs/decisions/0002-extract-plugins-infrastructure-from-edx-platform.rst
+
+Enable Plugins in an IDA
+------------------------
+
+See `instructions to enable plugins for an IDA`_.
+
+.. _instructions to enable plugins for an IDA: https://github.com/openedx/edx-django-utils/blob/master/edx_django_utils/plugins/docs/how_tos/how_to_enable_plugins_for_an_ida.rst
```

Comment:
I believe this is the right way to make this a proper doc link that will work on Read The Docs:

```suggestion
See :doc:`instructions to enable plugins for an IDA <how_to_enable_plugins_for_an_ida>`.
```